### ISSUE TITLE
`rumdl fmt` rewrites MkDocs `!!!` admonition bodies into fenced code blocks

### ISSUE THREAD

**Issue by pygarap** [rvben/rumdl#346]
## Summary

When formatting MkDocs-style admonitions (for example `!!! note`), `rumdl fmt` converts the required indented admonition body into a fenced code block (often ```text), which breaks the rendered output.

## Repro

1. Create `repro.md`:

```markdown
!!! note

    This approach shares state between the composited efforts. This means that authentication, database pooling, and other things will be usable between components. The [API reference](../../api/routing.md) for this documentation displays options for more controls like Router-specific lifespans, URL prefixes, and more.
```

2. Use this config (or equivalent):

```toml
[global]
flavor = "mkdocs"

[MD046]
style = "fenced"

[MD048]
style = "backtick"
```

3. Run:

```sh
rumdl fmt repro.md
```

## Actual result

The admonition body is rewritten as a top-level fenced code block:

````markdown
!!! note

```text
This approach shares state between the composited efforts. This means that authentication, database pooling, and other things will be usable between components. The [API reference](../../api/routing.md) for this documentation displays options for more controls like Router-specific lifespans, URL prefixes, and more.
```
````

## Expected result

The indented body should remain admonition content (or, at minimum, `rumdl` should avoid fixes that change the semantics of MkDocs admonitions in `flavor = "mkdocs"`).

## Environment

- `rumdl` [v0.1.3](https://github.com/rvben/rumdl/releases/tag/v0.1.3)
- `flavor = "mkdocs"`
- Rules involved: `MD046`, `MD048`

**Comment by rvben** [comment]
The fix for this was included in v0.0.212 (January 7), which predates this issue. I tested your exact repro case on v0.1.5 and the admonition content is correctly preserved - it's not being converted to a fenced code block.

Could you verify on v0.1.5? If you're still seeing the issue, please share the exact file that's causing problems - there might be an edge case I'm not catching in testing.
